sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/TestService.java

```diff
@@ -132,9 +139,31 @@ public Test startTest(Test test, List<String> jiraIds, String configXML) {
 
     @Transactional(rollbackFor = Exception.class)
     public void createTest(Test test) {
+        validateTestFieldsLength(test);
         testMapper.createTest(test);
     }
 
+    private void validateTestFieldsLength(Test test) {
```

Comment:
No plans to log field length as well? Please add a comment explicitly saying why this logging is added and when it should be gone (todo item maybe)